test/examples/ucx_hello_world.h

```diff
@@ -7,13 +7,24 @@
 #ifndef UCX_HELLO_WORLD_H
 #define UCX_HELLO_WORLD_H
 
+#include <ucs/memory/memory_type.h>
+
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
 #include <unistd.h>
 #include <netdb.h>
 
+#ifdef UCX_HELLO_WORLD_CUDA
+# include <cuda.h>
+# include <cuda_runtime.h>
```

Comment:
Remove space between `#` and `include`.

Reply:
typically we use 2 spaces in such case:)
(in `ib_verbs.h` for example)

Reply:
Sorry, I wasn't aware of this formatting.